kasa/cli/light.py

```diff
@@ -131,7 +131,11 @@ def presets_list(dev: Device):
         return
 
     for preset in light_preset.preset_states_list:
-        echo(preset)
+        echo(
+            f"[{preset.index}] Hue: {preset.hue:3}  Saturation: {preset.saturation:3}  "
```

Comment:
```suggestion
            f"[{index}] Hue: {preset.hue:3}  Saturation: {preset.saturation:3}  "
```
Maybe? I guess other presets commands do not work for tapo devices at the moment and we should add checks to error out on unsupported devices for those.